src/libs/joystick/manager.ts

```diff
@@ -32,6 +32,7 @@ const JoystickMapVidPid: Map<string, JoystickModel> = new Map([
   ['054c:0ce6', JoystickModel.DualSense],
   ['054c:09cc', JoystickModel.DualShock4],
   ['045e:02e0', JoystickModel.XboxOneS_Bluetooth],
+  ['045e:02fd', JoystickModel.XboxOneS_Bluetooth],
```

Comment:
@rafaellehmkuhl I have a xbox one s controller but it showed up different on my machine. (ubuntu 20.22)

Reply:
From what I'm checking [here](https://devicehunt.com/view/type/usb/vendor/045E/device/02E0) we are actually using the "Xbox One Controller Bluetooth" PID as an "S" version here, which it isn't.

Could you keep your change and also change the `XboxOneS_Bluetooth` to `XboxOne_Bluetooth`?

Reply:
@rafaellehmkuhl I checked the website and my controller being "Xbox One S Controller [Bluetooth]" is correct. I'll be interested to see what you all have.

Reply:
Yup, I'm agreeing with you haha

I think one of us has an Xbox One controller, thought it was an S one, but was actually a regular One, and we added it incorrectly.